component-sets/default-components/styles/_chart.scss

```diff
@@ -0,0 +1,113 @@
+.chart {
+    margin: 12px auto;
+    position: relative;
+    clear: both;
+
+    iframe {
```

Comment:
This `iframe` should be `div` as it will also hold placeholders, in Studio we will add the iframe, once the drop event happens.
Otherwise some styles like 500px height are not taken into account when rendering in Studio
<img width="836" alt="image" src="https://github.com/WoodWing/csde-components-boilerplate/assets/29701574/ae14e130-c000-4174-9a08-9a14c7a62439">


I did the change because thats how I had it locally and it works:
<img width="1262" alt="image" src="https://github.com/WoodWing/csde-components-boilerplate/assets/29701574/47770dd8-c50f-494d-a921-82e7c09bdcf6">

Reply:
Whoops, forgotten to include the styles when changing the iframe to a div. Will do!